SnapWar/Themes/Color.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Colors.swift
+//  SnapWar
+//
+//  Created by Pia Leung on 06/04/2020.
+//  Copyright © 2020 stickerwar. All rights reserved.
+//
+
+import UIKit
+
+struct Color {
```

Comment:
also see people using `enums` lol lmk if you feel strongly about one or the other

Reply:
i think enum is better, there are some examples for using enums :https://github.com/davidozhang/spycodes/blob/master/Spycodes/Core/SCStates.swift

Reply:
even cleaner: https://docs.swift.org/swift-book/LanguageGuide/Enumerations.html

Reply:
looks a little ugly to do with the colors case https://stackoverflow.com/q/49192068